benchmark/inference/inference_benchmark.py

```diff
@@ -225,6 +229,15 @@ def run(args: argparse.ArgumentParser):
                         print(f'Throughput: {throughput:.3f} samples/s')
                         print(f'Latency: {latency:.3f} ms')
 
+                        if args.write_csv:
```

Comment:
I think there is no real reason to gate this behind `if args.write_csv`. I would just gate the final `write_to_csv` call.

Reply:
Done.